embeddings/task/flair/flair_task.py

```diff
@@ -0,0 +1,92 @@
+import abc
```

Comment:
Agree with this opinion. I think it could be restricted by __init__. But the naming of task should be specific instead. Generally I think is a good idea to 3rdparty-specific functionality if it is above one file to keep in seperate modules. 

@riomus ?

Reply:
@albert097 renamed this modules to convention `{library_name}_{module}` e.g. `embeddings/task/flair_task/flair_task.py` and apllied this convention to other modules. Should this be our final convetion?